### PR TITLE
RavenDB-14680 Compare Exchange Tombstones are being deleted after ful…

### PR DESCRIPTION
**Pull request by efratshenhar**
…l backups only

### CODE REVIEW

File: src/Raven.Server/ServerWide/Maintenance/ClusterObserver.cs [hunk 1]
```diff
@@ -453,7 +453,10 @@ private long GetMaxTombstonesEtagToDelete(TransactionOperationContext context, s
                     || lastRaftIndexBlittable.TryGet(nameof(PeriodicBackupStatus.LastEtag), out long? lastRaftIndex) == false)
                     continue;
 
-                if (lastIncrementalBackupDate == null || lastRaftIndex == null)
+                if (localBackup.TryGet(nameof(PeriodicBackupStatus.LastFullBackup), out DateTime? lastfullBackupDate) == false)
```
Comment (karmeli87) [inline src/Raven.Server/ServerWide/Maintenance/ClusterObserver.cs:456]: I think we would want to have an identical code to 5.0 here.
cherry-picking will be problematic here, so simply copy-paste?

note that in 5.0 we don't check for `LastIncrementalBackup` at all.
@ppekrol thoughts?

Comment (ppekrol) [reply src/Raven.Server/ServerWide/Maintenance/ClusterObserver.cs:456]: Is this code problematic? What case is not covered?

Comment (efratshenhar) [reply src/Raven.Server/ServerWide/Maintenance/ClusterObserver.cs:456]: If we have only full backup

**Comment by ppekrol** [comment]
Handled here: https://github.com/ravendb/ravendb/pull/11034

**Pull request closed without merging**